### PR TITLE
[15 min fix] Fix following tags input a11y errors

### PR DESCRIPTION
**Pull request by aitchiss**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [X] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

The Dashboard > Following Tags page had a few accessibility errors where inputs were not given a unique id or a label. This PR fixes both of those issues.

## Related Tickets & Documents

N/A

## QA Instructions, Screenshots, Recordings

- Go to the user dashboard
- Select the 'Following Tags' link
- Check with a screen reader that each of the inputs has a unique/logical name
- Check that you can adjust the weights, and save the form
- Check that when you refresh/revisit the page, the weights are populated with the new values

Axe output before the change:

screenshot showing 9 axe errors

Axe output after the change:

screenshot showing 3 axe errors

Screenshot of VoiceOver rotor form controls (after the change):

screenshot showing the list of inputs with their names

### UI accessibility concerns?

This improves the accessibility of this section of the dashboard, allowing screen reader users to know what each input is for 

## Added tests?

- [ ] Yes
- [X] No, and this is why: No existing tests were affected, and small tweaks like these would be better tested as part of a larger e2e critical flow test
- [ ] I need help with writing tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [ ] I've updated the [Developer Docs](https://docs.forem.com) and/or
      [Admin Guide](https://forem.gitbook.io/forem-admin-guide/), or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] I've updated the README or added inline documentation
- [X] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [ ] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [ ] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

### CODE REVIEW

File: app/assets/javascripts/initializers/initScrolling.js [hunk 1]
```diff
@@ -103,32 +103,17 @@ function buildTagsHTML(tag) {
...
+    <h3 class="s:mb-1 p-0 fw-medium">
+      <a href="/t/${tag.name}" class="crayons-tag crayons-tag--l">
+        <span class="crayons-tag__prefix">#</span>${tag.name}
+      </a>
+      ${antifollow}
+    </h3>
+    <p class="grid-cell__summary truncate-at-3"></p>
+    <input name="follows[][id]" id="follow_id_${tag.name}" type="hidden" form="follows_update_form" value="${tag.id}">
+    <input step="any" class="crayons-textfield flex-1 fs-s" required="required" type="number" form="follows_update_form" value="${tag.points}" name="follows[][explicit_points]" id="follow_points_${tag.name}" aria-label="${tag.name} tag weight">
+  </div>`;
```
Comment (rhymes) [inline app/assets/javascripts/initializers/initScrolling.js:116]: OMG thank you ❤️

**Review by rhymes — APPROVED** [review]
This is great!

BTW: I get way more a11y errors on DEV

 
Screenshot 2021-04-23 at 3 02 15 PM

**Comment by aitchiss** [comment]
> BTW: I get more a11y errors on DEV

If you unfollow all your tags you'll get fewer 😆  (one for every tag at the moment)

**Comment by rhymes** [comment]
@aitchiss ahhaha sure! but the 26 "form elements must have labels" have zero instances in your example and you follow a few.

**Review by jacobherrington — APPROVED** [review]
:+1:

**Pull request merged**